content/en/docs/ops/configuration/telemetry/telemetry-v1-vs-v2/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Differences between Telemetry v1 and v2
+description: Tradeoffs and gaps between Telemetry v1 and v2.
+weight: 10
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Providing uniform telemetry and extension mechanisms to extend the functionality
+in Istio have been the driving factors for the architectural choices made in
+Istio. From early days, mixer aka Telemetry v1 provided both of these
+functionalities where an out-of-process service gathered attributes from the
+workload Envoy proxies, combined it with platform data like Kubernetes services
+and deployment associations and generated rich telemetry data like Prometheus
+metrics, tracing and logs. It also provided an extension mechanism where
+users could write adapters to ingest these attributes and enforce policies or
+transform them before exporting it to a third-party monitoring tool.
+
+While this provided a flexible architecture, mixer consumed significant resources
+in clusters with large amounts of workloads and often became a single point of failure.
```

Comment:
A link to the set of issues within the text would greatly re-enforce this idea. I'm sure there were plenty of other issues fixed.

https://github.com/istio/istio/issues/11400
https://istio.io/latest/blog/2019/performance-best-practices/#istio-1-2-performance